GliaWidgets/Glia.swift

```diff
@@ -1,35 +1,147 @@
 import UIKit
 import SalemoveSDK
 
+/// Engagement media type.
 public enum EngagementKind {
+    /// No engagement
     case none
+    /// Chat
     case chat
+    /// Audio call
     case audioCall
+    /// Video call
     case videoCall
 }
 
+/// An event providing engagement state information.
 public enum GliaEvent {
+    /// Session was started
     case started
+    /// Engagement media type changed
     case engagementChanged(EngagementKind)
+    /// Session has ended
     case ended
+    /// Engagement window was minimized
     case minimized
+    /// Engagement window was maximized
     case maximized
 }
 
+/// Used to provide `UIWindowScene` to the framework.
 public protocol SceneProvider: AnyObject {
     @available(iOS 13.0, *)
     func windowScene() -> UIWindowScene?
 }
 
+/// Glia's engagement interface.
+///
+/// ## Integration
```

Comment:
Maybe question to Henrik (@henrikver) - how do we plan to distribute this SDK?
I ask to understand if this Integration here, should:
1. ...stay in this swift file, to be proper part of the distributed package.
2. ...be copied and/or moved to README.md file, which is kind of a standard in case of GitHub repo's (and other utilities and code snippets as well, be it README.md or README.txt or just README, meaning is the same).
3. ...stay here in this swift file, but also be copied to our own dev docs environment in ReadMe.

Follow-up - depending on where these instructions will be located, the guidelines regarding image types, and default values, that we have discussed in the comments here, can find their place under the sun.

Reply:
The Widgets SDK will be distributed in the same way as Core SDK, so the documentation should also follow this.